src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * The main service provider for the version 2 of the Widgets.
+ *
+ * @since   TBD
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+
+namespace Tribe\Events\Views\V2\Widgets;
+
+use Tribe\Events\Views\V2\Views\Widgets\List_Widget_View;
+use Tribe\Events\Views\V2\Views\Widgets\Widget_List_View;
+
+/**
+ * Class Service_Provider
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Widgets
+ */
+class Service_Provider extends \tad_DI52_ServiceProvider {
+
+	/**
+	 * Variable that holds the name of the widgets being created
+	 *
+	 * @since TBD
+	 *
+	 * @var array
```

Comment:
Here and elsewhere, define the type of values part of the array:

```suggestion
	 * @var array<string>
```